Big-Spender/readme.md

```diff
@@ -69,22 +74,28 @@ INSERT YOUR QUERY HERE
 
 ```sql
 INSERT YOUR QUERY HERE
-```
+select * from spends where description ilike '%fee%';
+```  
 
-**Farnoosh:** Hi, it's me again. It turns out we also need the transactions that have the expense area of 'Better Hospital Food'. Can you help us with that one?
+**Farnoosh:** Hi, it's me again. It turns out we also need the transactions that have the expense "expense_areas" area of 'Better Hospital Food'. Can you help us with that one?
 
 **You:** No worries. Here's the query for that:
 
 ```sql
 INSERT YOUR QUERY HERE
+  select * from spends join expense_areas on expense_areas.id = spends.expense_area_id 
+big_spender-# where expense_areas.id = 2;   
 ```
 
-**Claire:** Great, that's very helpful. How about the total amount spent for each month?
+**Claire:** Great, that's very helpful. How about the total amount       ?
 
 **You:** You can get that by using the GROUP BY clause. Here's the query:
 
 ```sql
 CREATE YOUR QUERY HERE
+
+select sum(spends.amount) from spends join expense_areas on expense_areas.id = spends.expense_area_id                                                
+where expense_areas.id = 2;
```

Comment:
How would we be able to get this total for each month? This would give us the total spent for Better Hospital Foods, but what about per month?

Reply:
sure 
select select to_char(spends.date,'Mon') as mon, sum(spends.amount) 
from spends join expense_areas on (expense_areas.id = spends.expense_area_id)                                                                where expense_area ilike '%Better Hospital Food%' 
group by mon;